### PR TITLE
Add ccb support

### PR DESCRIPTION
**Pull request by slahabar**

### CODE REVIEW

File: DataScience/Experimentation.py [hunk 1]
```diff
@@ -200,14 +200,19 @@ def parse_cb_types(val):
...
+        return policy_command.replace('--ccb_explore_adf --epsilon 0', '--ccb_explore_adf --epsilon 0.2')
+    else:
+        return policy_command.replace('--cb_adf', '--cb_explore_adf --epsilon 0.2');
 
+def generate_predictions_files(log_fp, policies, log_type):
     predictions_files = []
     data = {}
     data['policies'] = []
-    Logger.info('Generating predictions files (using --cb_explore_adf) for {} policies:'.format(len(policies)))
+    Logger.info('Generating predictions files for {} '.format(log_type) + 'for {} policies:'.format(len(policies)));
```
Comment (cheng-tan) [inline DataScience/Experimentation.py:213]: remove `;`

File: DataScience/ds_parse.py [hunk 2]
```diff
@@ -146,6 +146,8 @@ def json_cooked(x, do_devType=False, do_VWState=False, do_p_vec=False):
     ind6 = x.find(b'"',ind5)
     ind7 = x.find(b',"a"',ind5)
     ind8 = x.find(b']',ind7+7)          # equal to: x.find('],"c',ind7+8)
+    if ind8 == -1:
```
Comment (cheng-tan) [inline DataScience/ds_parse.py:149]: when will it run into this case?

**Review by cheng-tan — COMMENTED** [review]
Looks good

File: DataScience/Experimentation.py [hunk 3]
```diff
@@ -200,14 +200,19 @@ def parse_cb_types(val):
         raise argparse.ArgumentTypeError('Input "{}" is an invalid cb_types input string - Valid cb_types are mtr, dr, ips'.format(val))
     return cb_types
 
-def generate_predictions_files(log_fp, policies):
+def create_prediction_command(policy_command, log_type):
+    if log_type == 'ccb':
+        return policy_command.replace('--ccb_explore_adf --epsilon 0', '--ccb_explore_adf --epsilon 0.2')
```
Comment (homezcx) [inline DataScience/Experimentation.py:205]: just curious: why one has `--epsilon 0` but the other does not have it.

Comment (cheng-tan) [reply DataScience/Experimentation.py:205]: cb has `cb_adf` and `cb_explore_adf`, when sweeping the commands to find the best policies, we used `cb_adf` and for prediction we used `cb_explore_adf --epsilon 0.2`

ccb only has `ccb_explore_adf`, when sweeping we used `ccb_explore_adf --epsilon 0` to do exploitation and for prediction we used `ccb_explore_adf --epsilon 0.2`

File: DataScience/ds_parse.py (hunk 2 above)
Comment (slahabar) [reply DataScience/ds_parse.py:149]: Probably bad formatted data. Not sure though. I pulled this fix in from master.

File: DataScience/Experimentation.py (hunk 1 above)
Comment (slahabar) [reply DataScience/Experimentation.py:213]: I'll fix this in the next PR coming shortly.

**Review by ataymano — APPROVED** [review]

**Review by cheng-tan — APPROVED** [review]

**Pull request merged**